omnigibson/robots/robot_base.py

```diff
@@ -2,6 +2,8 @@
 from copy import deepcopy
 import numpy as np
 import matplotlib.pyplot as plt
+from telemoma.human_interface.teleop_core import TeleopAction
```

Comment:
Don't import this here. Creates an unnecessary dependency. Remove the type hint it's used for.